packages/ui/src/views/pull-request/pull-request-list-page.tsx

```diff
@@ -15,12 +15,7 @@ export interface PullRequestListProps {
   repoMetadata?: RepoRepositoryOutput
 }
 
-const PullRequestListPage: React.FC<PullRequestListProps> = ({
-  usePullRequestStore,
-  spaceId,
-  repoId,
-  repoMetadata
-}) => {
+const PullRequestList: React.FC<PullRequestListProps> = ({ usePullRequestStore, spaceId, repoId, repoMetadata }) => {
```

Comment:
Looks like `React` isn't in this file; can we add `import { FC } from 'react'`?

Reply:
done